### DISCUSSION TITLE
获取最新评论？

### DISCUSSION THREAD

**Discussion by jerryc127** [ArtalkJS/Artalk#205] [Q&A] [answered]
怎么获取最新评论？

看了文档似乎没有提供这个功能

api 只能获取指定文章的评论。。。

**Comment by qwqcode** [comment, marked as answer]
代码可以参考：https://github.com/ArtalkJS/ArtalkGo/blob/master/http/stat.go#L56

API 文档还没来得及写 🤣 以后再写

POST 请求 `/api/stat`

携带参数：

- `type: "latest_comments"`
- `site_name: "你的站点名"`
- `limit: 20`

**Reply by jerryc127** [reply]
thx
